index.html

```diff
@@ -61,10 +80,11 @@ <h4 class="h4-connect">LET'S CONNECT</h4>
   
 
     <!-- work one section -->
-
+    <section id="Portfolio">
      <section class="card-works works">
       <div class="image ">
-        <img src="images/Snap.png" alt="Card1"/>
+        <img class="desktop-image" src="images/Snap.png" alt="Project"/>
+        <img class="mobile-image" src="images/Snapshoot1.png" alt="Project"/>
       </div>
```

Comment:
- [ ] Your popup has good content, beautiful pictures, and good colors. But if possible you can improve the design by fixing the size of your images and the buttons, according to the [templet](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A1143) :+1:

Reply:
Thanks. I have looked and adjusted them